### DISCUSSION TITLE
一些希望主题添加的功能

### DISCUSSION THREAD

**Discussion by ZheWana** [Tokinx/Wing#112] [想法]
- 希望添加首页顶部“笔记”栏名称的自定义
- 希望添加友链是否显示头像的可选项

**Comment by ZheWana** [comment]
### 补充功能：
* 希望添加自定义首次进入博客时使用的主题

### 补充瑕疵：
* Light模式下文章内目录看不清

**Comment by LingJueYa** [comment]
希望添加大图组图模式呀
